### PR TITLE
remove welsh feature flag

### PR DESCRIPTION
**Pull request by wp94**

### CODE REVIEW

File: app/views/templates/Layout.scala.html [hunk 1]
```diff
@@ -49,13 +49,11 @@
 }
 
 @beforeContent = {
-    @if(appConfig.languageTranslationEnabled) {
         @hmrcLanguageSelect(LanguageSelect(
             if (messages.lang.code == "cy") Cy else En,
             (En, routes.LanguageSwitchController.switchToLanguage("en").url),
             (Cy, routes.LanguageSwitchController.switchToLanguage("cy").url)
         ))
```
Comment (PatrykRudnicki) [inline app/views/templates/Layout.scala.html:56]: This whole element needs to be removed too, because it's about the Welsh component

**Review by dips7189 — APPROVED** [review]

File: app/views/templates/Layout.scala.html (hunk 1 above)
Comment (wp94) [reply app/views/templates/Layout.scala.html:56]: Thanks Patryk, I'll remove this too

**Review by PatrykRudnicki — APPROVED** [review]

File: app/views/templates/Layout.scala.html (hunk 1 above)
Comment (PatrykRudnicki) [reply app/views/templates/Layout.scala.html:56]: My misunderstanding! It's a correct part and need to stay here

Comment (wp94) [reply app/views/templates/Layout.scala.html:56]: After discussion, this element will stay in

**Pull request merged**